api/gr_api.h

```diff
@@ -25,11 +25,12 @@ struct gr_api_response {
 #define EVENT_TYPE(module, id) (((uint32_t)(0xffff & module) << 16) | (0xffff & id))
 #define EVENT_TYPE_ALL UINT32_C(0xffffffff)
 
-#define GR_DEFAULT_SOCK_PATH "/run/grout.sock"
+#define GR_SOCK_NAME "\0grout.sock"
+#define GR_SOCK_NAME_LEN (sizeof(GR_SOCK_NAME) - 1)
```

Comment:
Why does this need to start with a NULL byte? It would be more convenient if this was handled in the function where the abstract socket name is actually used.